### ISSUE TITLE
Microsoft.EntityFrameworkCore.Sqlite 1.0.1 Migration Fails

### ISSUE THREAD

**Issue by nicojmb** [dotnet/efcore#6693]
### Steps to reproduce

From nuget console type : Add-Migration MyFirstMigration
### The issue

i've updated "Microsoft.EntityFrameworkCore.Sqlite" to 1.0.1 and when add or remove migration get this error:

```
Remove-Migration : Exception calling "CreateInstanceAndUnwrap" with "8" argument(s): "Could not load file or assembly 'Microsoft.EntityFrameworkCore, Version=1.0.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60' or one of its dependencies. The located assembly's manifest definition does not match the 
assembly reference. (Exception from HRESULT: 0x80131040)"
At line:1 char:1
+ Remove-Migration
+ ~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Remove-Migration], MethodInvocationException
    + FullyQualifiedErrorId : FileLoadException,Remove-Migration
```
### Further technical details

``` JSON
{
  "dependencies": {
    "Microsoft.EntityFrameworkCore.Sqlite": "1.0.1",
    "Microsoft.EntityFrameworkCore.Tools": "1.0.0-preview2-final",
    "Microsoft.NETCore.UniversalWindowsPlatform": "5.2.2",
    "Microsoft.Toolkit.Uwp": "1.1.0",
    "Microsoft.Toolkit.Uwp.Notifications": "1.1.0",
    "Microsoft.Toolkit.Uwp.UI.Controls": "1.1.0"
  },
  "frameworks": {
    "uap10.0": {}
  },
  "runtimes": {
    "win10-arm": {},
    "win10-arm-aot": {},
    "win10-x86": {},
    "win10-x86-aot": {},
    "win10-x64": {},
    "win10-x64-aot": {}
  }
}
```

EF Core version: (found in project.json or packages.config)
Operating system: 10 Aniversary Update
Visual Studio version: VS Community 2015 Update 3

**Comment by divega** [comment]
Duplicate of #5471. 

See https://github.com/aspnet/EntityFramework/issues/6551#issuecomment-247840623 for complete workaround.

**Comment by ramy-s236** [comment]
Hi, I am having the same issue. I have been using the fix described above successfully for months. I started a new project and tried setting up Entity and creating a migration the same way, but got the same error as op. I checked an older project and I could create a migration. The difference between the projects is the older project didn't have an update to Microsoft.EntityCore.Sqlite. The proposed solution alone does not fix the problem,

**Comment by ramy-s236** [comment]
I have the same problem as OP. I have been successfully using the given solution for months. I recently started a new project and set up ENtity in the same way and received the same error as OP. I checked an older project (that employs the given solution) and I could migrate. The difference is that the new project had Package Microsoft.EntityFrameworkCore.Sqlite version 1.0.1 installed and the older had version 1.0.0. Reverting to older version fixed the problem.
